CHANGES.md

```diff
@@ -92,7 +92,7 @@
 
 * ([#X](https://github.com/apache/beam/issues/X)).
 
-# [2.56.0] - Unreleased
```

Comment:
add date ( - 2024-05-01) ?

realize 2.55.1 also didn't have date, before that each section was dated

Reply:
Good call, fixed